py4syn/epics/ImageHDFClass.py

```diff
@@ -0,0 +1,111 @@
+"""Image HDF Class
+
+Python Class for Save HDF images .
+
+:platform: Unix
+:synopsis: Python Class for HDF images.
+
+.. moduleauthor:: Gabriel Fedel <gabriel.fedel@lnls.br>
+.. note:: 06/02/2017 [gabrielfedel]  first version released
+"""
+import os
+
+import numpy as np
+import h5py
+from py4syn.epics.StandardDevice import StandardDevice
+from py4syn.epics.ICountable import ICountable
+
+class ImageHDF(StandardDevice, ICountable):
```

Comment:
ImageHDF should be just a class. Your devices must be StandardDevice and ICountable.
E.g.: DxpFake(StandardDevice, ICountable, ImageHDF).
Do you agree?